packages/native/src/components/LinearGradient.tsx

```diff
@@ -24,14 +24,14 @@ const LinearGradientComponent = ({
   startY = 0,
   endX = 100,
   endY = 100,
-  style,
+  style: propStyle,
 }: LinearGradientComponentProps) => {
-  const { viewStyles } = extractStyles(style);
+  const { viewStyles: style } = extractStyles(propStyle);
```

Comment:
in this specific case you don't need to use extractStyles since there is no other styles being applied anywhere else.

Cases where extractStyles is useful:

- FormRow, where we must separate viewStyle from textStyle. We separate the two because they're applied to two separate elements:
https://github.com/draftbit/react-native-jigsaw/blob/a421be23d58d60b71257a16eb8db11d13a4713bb/packages/core/src/components/FormRow.tsx#L25